data/category-porn

```diff
@@ -3208,6 +3208,7 @@ xnostars.com
 xnxx-com.cfd
 xnxx-sex-videos.com
 xnxx-teens.com
+
```

Comment:
why

Reply:
I have no idea sometimes it will leave empty content when I input the domain.